src/content/posts/new-wireguard-mode/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "A more user-friendly transparent mode, based on WireGuard"
+summary: | 
+    The next release of mitmproxy is slated to ship with a new alternative to the existing "transparent" mode, based on
+    WireGuard. The new "wireguard" mode was designed to be much simpler to set up and use, and in most cases, it does not
+    require taking any actions with elevated privileges.
+date: 2022-09-23
+weight: 10
+tags: [tech]
+author:
+    name: Fabio Valentini
+    twitter: decathorpe
+---
+
+The next release of mitmproxy is slated to ship with a new alternative to the existing `transparent` mode, based on
+WireGuard. The new `wireguard` mode was designed to be much simpler to set up and use, and in most cases, it does not
+require taking any actions with elevated privileges.
+
+Connecting another device and routing its traffic through mitmproxy becomes as simple as connecting to a WireGuard VPN
+--- either by using the WireGuard configuration that is printed by mitmproxy, or by scanning the QR code that is shown
+in the mitmweb interface.
```

Comment:
A screenshot would be great! Or even a 30sec gif/video from a phone scanning the QR code from mitmweb and it auto-configured the wireguard phone app.

Reply:
Video I recorded the other day:
https://user-images.githubusercontent.com/1019198/192087170-9bf30c93-a2cd-4c24-8075-78963f4bb567.mp4

Screenshot:
![image](https://user-images.githubusercontent.com/1019198/192087189-797b0844-5586-4510-a564-e28a27561042.png)

Reply:
Great, I'll try to embed these two files :)